src/Symfony/Component/Console/Question/ChoiceQuestion.php

```diff
@@ -32,6 +32,10 @@ class ChoiceQuestion extends Question
      */
     public function __construct($question, array $choices, $default = null)
     {
+        if (!$choices) {
+            throw new \LogicException('Choice question must have at least 1 choice available.');
```

Comment:
changing it now would be BC break ;) anyway we should be good. If it is a domain level exception I would expect it to be thrown on count < 2 as  one option does not leave anything to choose from either ;)